entity_parser_key_parser_test.go

```diff
@@ -526,8 +536,49 @@ func TestEntityParse(t *testing.T) {
 					},
 				},
 			},
+			ETL:   EtlOff,
 			Error: nil,
 		},
+		{
+			Tag:       "name=jj primaryKey=ok, etl=on",
+			TableName: "jj",
+			PrimaryKey: &PrimaryKey{
+				PartitionKeys:  []string{"ok"},
+				ClusteringKeys: nil,
+			},
+			Error: nil,
+			ETL:   EtlOn,
+		},
+		{
+			Tag:       "name=jj primaryKey=ok etl=off",
```

Comment:
Add tests for different cases of etl value. "ON/OFF" and "On/Off"

Reply:
done